server/api/bot/submit.post.ts

```diff
@@ -5,7 +5,16 @@ const submitBotCodeSchema = z.object({
   code: z.string(),
 });
 
+const rateLimit = new Map<string, number>();
+const RATE_LIMIT_THRESHOLD_IN_MILLIS = 4000;
```

Comment:
nit: мы используем MS в проекте

```suggestion
const RATE_LIMIT_THRESHOLD_IN_MS = 4000;
```